tests/std/tests/VSO_0000000_regex_use/test.cpp

```diff
@@ -669,6 +669,20 @@ void test_gh_5160() {
     neg_regex.should_search_fail(L"xxxYxx\x2009xxxZxxx"); // U+2009 THIN SPACE
 }
 
+void test_gh_5167() {
+    // GH-5167: Limit backreference parsing to single digit for basic regular expressions
+    g_regexTester.should_match("abab0", R"x(\(ab*\)\10)x", basic);

```

Comment:
Style: As the raw string literal doesn't contain double quotes, the extra `x` delimiter is unnecessary and adds visual noise. Other raw string literals in this test don't do this.